### ISSUE TITLE
feat: live-update cell highlighting

### ISSUE THREAD

**Issue by jimniels** [quadratichq/quadratic#1317]
When you reference cells using `getCells` you have to _run_ the code to see the dashed outline highlighting what cells are being read.

https://github.com/quadratichq/quadratic/assets/1316441/1d8d345d-ba2e-4b5f-b66c-7062671f8ec7

Would be a much better experience if we could highlight this _as you changed the cell reference_. Otherwise the only way to know you've selected the right thing is to run the code first

**Comment by ddimaria** [comment]
@jimniels since there are cases where simple regex parsing of the values doesn't work (e.g. when the coords are variables), we decided to only show highlighted cells after a code run (via cells_accessed).

**Comment by jimniels** [comment]
Makes sense — but I still want it ha.

I would expect the software to be smart enough to figure out that if the values are static then it'll highlight them as I change it.

But if they're dynamic (i.e. variables) getting no highlights would feel expected.

**Comment by davidkircos** [comment]
Yeah I agree with Jim, but don't think this is really a high priority atm
